app/assets/javascripts/modules/track-brexit-qa-choices.js

```diff
@@ -4,33 +4,36 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 (function (global, GOVUK) {
   'use strict'
 
-  var $ = global.jQuery
-
   GOVUK.Modules.TrackBrexitQaChoices = function () {
     this.start = function (element) {
-      track(element)
+      track(element[0])
     }
 
     function track (element) {
-      element.on('submit', function (event) {
-        var $checkedOption, eventLabel, options
-        var $submittedForm = $(event.target)
-        var $checkedOptions = $submittedForm.find('input:checked')
-        var questionKey = $submittedForm.data('question-key')
-
-        if ($checkedOptions.length) {
-          $checkedOptions.each(function (index) {
-            $checkedOption = $(this)
-            var checkedOptionId = $checkedOption.attr('id')
-            var checkedOptionLabel = $submittedForm.find('label[for="' + checkedOptionId + '"]').text().trim()
+      element.addEventListener('submit', function (event) {
+        var eventLabel, options
+        var submittedForm = event.target
+        var checkedOptions = submittedForm.querySelectorAll('input:checked')
+        var questionKey = submittedForm.getAttribute('data-question-key')
+
+        if (checkedOptions.length) {
+          for (var i = 0; i < checkedOptions.length; i++) {
+            var checkedOptionId = checkedOptions[i].getAttribute('id')
+            var checkedOptionLabelText = submittedForm.querySelector('label[for="' + checkedOptionId + '"]')
+            var checkedOptionLabel = ''
+
+            if (checkedOptionLabelText != null) {
```

Comment:
Normally the linter complains about `!=` instead of `!==` but the linters are specific to applications so this is probably fine. For interest here's a thread on the difference: https://stackoverflow.com/questions/1946063/in-javascript-is-same-as